qualtran/bloqs/state_preparation/sparse_state_preparation_via_rotations.py

```diff
@@ -58,6 +58,7 @@ class SparseStatePreparationViaRotations(Bloq):
     nonzero_coeffs: Union[tuple[complex, ...], HasLength] = field(converter=_to_tuple_or_has_length)
     N: SymbolicInt
     phase_bitsize: SymbolicInt
+    target_bitsize: SymbolicInt = field()
```

Comment:
Can you add a test that uses a bigger target_bitsize than default and passes?

Reply:
added a bloq example and test